src/checkers/inference/InferenceValidator.java

```diff
@@ -202,8 +202,9 @@ public Void visitDeclared(AnnotatedDeclaredType type, Tree tree) {
             // ParameterizedTypeTree));
             break;
         default:
-            System.err.printf("TypeValidator.visitDeclared unhandled tree: %s of kind %s\n",
-                            tree, tree.getKind());
+            // This caused too many error outputs
+//            System.err.printf("TypeValidator.visitDeclared unhandled tree: %s of kind %s\n",
```

Comment:
Can we using logger to still log this info? If it is too verbose, we can set the log level to a finer one. It seems better than just lost this info.

Reply:
If you see `BaseTypeValidator`, this line is also commented out.

Reply:
I would keep this to make it consistent.